src/App.ts

```diff
@@ -927,8 +927,8 @@ function isBodyWithTypeEnterpriseInstall(body: AnyMiddlewareArgs['body'], type:
     }
   }
   // command payloads have this property set as a string
-  if (body.is_enterprise_install === 'true') {
-    return true;
+  if (typeof body.is_enterprise_install === 'string') {
+    return body.is_enterprise_install === 'true';
```

Comment:
I'm confused here, if it is of type `string` and the value is `'false'`, wouldn't you be returning the incorrect value here?